### DISCUSSION TITLE
OS Ventura 13.0.1 and BlackHole

### DISCUSSION THREAD

**Discussion by Mikehike7** [ExistentialAudio/BlackHole#697] [Q&A] [answered]
This week, when BlackHole (setup as required for recording core iMac audio), actuating the record button caused Audacity to crash. Otherwise, Audacity worked normally for editing, just crashing when trying to record. Since this may have been the first time using OS Ventura with Audacity and BlackHole, I then updated Audacity and BlackHole to the latest versions. Now Audacity is stable and still works fine for editing, but is not able to record iMac audio. Actuating the Audacity record button works but there is no sound input to Audacity through BlackHole. I rechecked Audio MIDI and Audacity setups and they appear to be exactly as required. Thank you for your assistance.

iMac OS Ventura 13.0.1
BlackHole: v0.4.0
Audacity v3.2.3

Audio MIDI setup: 

Audio Devices: 
-BlackHole 2ch, 2 ins / 2 outs
-Headphones, 0 ins / 2 outs
>Multi-Output Device: Primary Device: BlackHole 2ch, Sample Rate 48.0kHz
Use column checked BlackHole 2ch, Drift Correction checked
Use column checked External Headphones 0 ins/2outs
, Drift Correction checked
-Blackhole 2ch
-External Headphones

Audacity setup: 

Audio SetupButton:  
-host: Core Audio. 
-playback device: Multi-Output Device
-recording device: BlackHole 2ch
-recording channels: 2 (stereo) recording channels
-audio settings: as stated above with added line under interface section: Using: PortAudio v19.7.0-devel, revision unknown

**Comment by devinroth** [comment]
Audacity crashes multiple times a day for me. To be fair I’m constantly adding and removing audio devices.

Other than that, BlackHole is working completely fine with Audacity. I suspect that Audacity doesn’t have mic permission on your computer. Check that first.

Devin

**Reply by Mikehike7** [reply]
When I bring up Audacity preferences, 
under Devices I see 
Interface: 
Host is Core Audio in dropdown with no other choices. Using: PortAudio v19.7.0-devel, revision unknown with no dropdown. 
Playback:
Device: Multi-Output Device in dropdown. 
Recording: 
Device:  BlackHole 2ch in dropdown. Channels 2 (Stereo) in dropdown. 

under MIDI Devices, I see
Interface: Host dropdown No MIDI interfaces with no other choices. Using: PortMidi. Under Playback is see Device No devices found in the dropdown with no other choices.
Playback:
Device: No devices found in dropdown with no other choices
MIDI Synth Latency (ms): 5 in dropdown

**Comment by devinroth** [comment, marked as answer]
• Check System Preferences -> Security & Privacy -> Privacy -> Microphone to make sure your digital audio workstation (DAW) has microphone access.

**Reply by Mikehike7** [reply]
Yes, system Microphone for Audacity was switched off. I switched it on and went back to Audacity and changed Record Device back to BlackHole and recording internal sound now works!. This system check should really be added to the Black Hole setup. THANK YOU Devin, you're the best!
Mike

**Comment by devinroth** [comment]
BlackHole has no control over that. It’s an Audacity issue. And yes I agree, they should be checking that. Please bug them about it.

Devin
